bin/setup_stack

```diff
@@ -116,7 +116,11 @@ for index in 1; do
     cp -r topo/$mode/faucet inst/$container/
     docker kill $container || true
     if [ -n "$fot" ]; then
+        mkdir -p inst/$container/faucet_br/faucet
+        cp -r topo/$mode/faucet_br/* inst/$container/faucet_br/faucet/
+        ls inst/$container/faucet_br/faucet/
```

Comment:
remove extra debugging for now (the ls here and two lines below)

Reply:
Changed.